app/shared/src/commonMain/kotlin/ui/subject/episode/video/sidesheet/EditDanmakuRegexFilterSideSheet.kt

```diff
@@ -0,0 +1,181 @@
+            "填写用于屏蔽的正则表达式，例如：‘.*签.*’ 会屏蔽所有含有文字‘签’的弹幕。"
+        } else {
+            "竖屏状态下无法编辑"
+        }
+    }
+
+    fun handleAdd(): Unit {
+        if (regexTextFieldValue.isNotBlank()) {
+            danmakuRegexFilterState.addDanmakuRegexFilter(
+                DanmakuRegexFilter(
+                    id = Uuid.randomString(),
+                    name = "",
+                    regex = regexTextFieldValue,
+                    enabled = true,
+                ),
+            )
+            regexTextFieldValue = "" // Clear the text field after adding
+            focusManager.clearFocus()
+            isError = false
+        } else {
+            isError = true
+        }
+    }
+
+    EpisodeVideoSettingsSideSheet(
+        onDismissRequest = onDismissRequest,
+        title = { Text(text = "正则弹幕过滤管理") },
+        closeButton = {
+            IconButton(onClick = onDismissRequest) {
+                Icon(Icons.Rounded.Close, contentDescription = "关闭")
+            }
+        },
+        modifier = modifier.testTag(TAG_EPISODE_SELECTOR_SHEET)
+            .clickable(onClick = { focusManager.clearFocus() }),
+    ) {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable { focusManager.clearFocus() },
+        ) {
+            Surface {
+                Column(
+                    modifier.padding(horizontal = 16.dp)
+                        .fillMaxWidth()
+                        .navigationBarsPadding()
+                        .verticalScroll(rememberScrollState()),
+                ) {
+                    // 输入框
+                    Column(
+                        modifier = Modifier.padding(top = 16.dp),
+                        verticalArrangement = Arrangement.spacedBy(16.dp),
+                    ) {
+                        OutlinedTextField(
+                            value = regexTextFieldValue,
+                            onValueChange = {
+                                regexTextFieldValue = it
+                                if (expanded) {
+                                    regexTextFieldOutlineTitleText =
+                                        "填写用于屏蔽的正则表达式，例如：‘.*签.*’ 会屏蔽所有含有文字‘签’的弹幕。"
+                                }
+                            },
+                            isError = isError,
+                            supportingText = {
+                                if (isError) {
+                                    Text(
+                                        modifier = Modifier.fillMaxWidth(),
+                                        text = "正则表达式不能为空",
+                                        color = MaterialTheme.colorScheme.error,
+                                    )
+                                }
+                            },
+                            label = {
+                                Text(
+                                    text = regexTextFieldOutlineTitleText,
```

Comment:
```suggestion
                                    text =  if (expanded) {
            "填写用于屏蔽的正则表达式，例如：‘.*签.*’ 会屏蔽所有含有文字‘签’的弹幕。"
        } else {
            "竖屏状态下无法编辑"
        }
```

直接写到这里面更好

记得 format 一下